packages/tokens/src/border.yaml

```
@@ -1,13 +1,13 @@
 border:
   radius:
     regular:
-      value: 0.188rem
+      value: '{ size.3.value }'
     medium:
-      value: 0.375rem
+      value: '{ size.6.value }'
   width:
     100:
-      value: 0rem
+      value: '{ size.0.value }'
     200:
-      value: 0.063rem
+      value: '{ size.0.value }'
```

```suggestion
      value: '{ size.1.value }'
```
Or was it intentionally the same as `100`?

---

oh jeez, nice catch!